client/globals.d.ts

```diff
@@ -85,6 +85,11 @@ declare global {
 		errorMessage: string;
 		onBoardingDisabled: boolean;
 		isBnplAffirmAfterpayEnabled: boolean;
+		connectIncentive?: {
+			id: string;
+			description: string;
+			tc_url: string;
```

Comment:
It is the promo/incentive dedicated Terms and Conditions (aka TC) page URL.

Reply:
We were mapping snake case to camel case before, but decided to update ESLint rules to allow the former, because that's what's coming from the server.

I prefer to be consistent, but I think it will need to automatically map every prop we receive and send to the server to ensure we don't mix them by mistake.

Reply:
Ah yes, I understand the mapping makes it difficult. Thanks for explaining!